dotnet/Liberator.Driver/RatDriverMethods.cs

```diff
@@ -322,16 +322,10 @@ public void ClickLinkAndWait(IWebElement element, [Optional, DefaultParameterVal
             try

```

Comment:
Im not sure we need the waitForTarget bool - reasoning: The method name implies that the user has already selected a true for the wait as the method name is ClickLinkAndWait. If the method name was just ClickLink then user might want to control if there is a wait or not - but ClickLinkAndWait implies that there is always going to be a wait. Thoughts?

Reply:
Changed